### ISSUE TITLE
Knowledge Base/Documentation Panel Indicator

### ISSUE THREAD

**Issue by lukas-frey** [GuavaCZ/filament-knowledge-base#12]
> > Hm, does it make sense to have the documentation button on the documentation panel itself?
> 
> That went through my head too. I felt it was a nice indicator that you're in the knowledge base. Do you feel it's confusing when it's in the knowledge base panel?

Personally I think it's odd to have it there since you're already there. I think some kind of indicator would fit better in the header.

_Originally posted by @gavinhewitt in https://github.com/GuavaCZ/filament-knowledge-base/issues/10#issuecomment-2098214610_

**Comment by lukas-frey** [comment]
> > > Hm, does it make sense to have the documentation button on the documentation panel itself?
> > 
> > 
> > That went through my head too. I felt it was a nice indicator that you're in the knowledge base. Do you feel it's confusing when it's in the knowledge base panel?
> 
> Personally I think it's odd to have it there since you're already there. I think some kind of indicator would fit better in the header.

Hmm, think an indicator is needed at all though? Is it confusing for you when you switch from the regular panel to the documentation panel?

**Comment by gavinhewitt** [comment]
> > > > Hm, does it make sense to have the documentation button on the documentation panel itself?
> > > 
> > > 
> > > That went through my head too. I felt it was a nice indicator that you're in the knowledge base. Do you feel it's confusing when it's in the knowledge base panel?
> > 
> > 
> > Personally I think it's odd to have it there since you're already there. I think some kind of indicator would fit better in the header.
> 
> Hmm, think an indicator is needed at all though? Is it confusing for you when you switch from the regular panel to the documentation panel?

It's not *really* needed but if you're going to provide some indicator the header would be better I think. Maybe add ' - Knowledge Base' (or something configurable) after the panel name?

**Comment by lukas-frey** [comment]
Good idea, you should be able to configure the title already of the panel, using:

```php
KnowledgeBasePanel::configureUsing(fn($panel) => $panel->brandName());
```

So maybe for now that might be enough.

Before the release I was playing with a around the inner screen of the window or a colored banner at the top of the panel. But then I removed it as I found it unnecessarily disturbing.

**Comment by gavinhewitt** [comment]
> Good idea, you should be able to configure the title already of the panel, using:
> 
> ```
> KnowledgeBasePanel::configureUsing(fn($panel) => $panel->brandName());
> ```

Ah yes, that works too 👍🏼

**Comment by lukas-frey** [comment]
From version 1.2.1 the button is now only in the panels where the plugin is enabled (not the KB panel itself anymore) and by default uses the name from the translations. It can be overridden as in the example above. :)

**Comment by DevRaeph** [comment]
Is there a way to go back to the normal panel? Like the documentation button in the bottom left of the side panel, but in reverse for the docs, i.e. a back button?

**Comment by lukas-frey** [comment]
@DevRaeph No, since the user can have multiple panels, I didn't want to add an unnecessary button that people might not need. The panel is extendable, so you can add it on your own.

Maybe if I can add a opt-in option for a Back to default panel button 🤔

**Comment by gavinhewitt** [comment]
> Is there a way to go back to the normal panel? Like the documentation button in the bottom left of the side panel, but in reverse for the docs, i.e. a back button?

Just open the docs in a new window :-)
